Automattic/WP-Job-Manager
# Feature: Reengaging customers.

Improves discoverability of WPJM updates and license activation experience.

### Changes proposed in this Pull Request
* #2379
  * #2383
  * #2385
* #2376
  * #2387
* #2375
  * #2390
  * #2394
    * #2392
  * #2396
    * #2397
  * #2400
* #2382
  * #2401
* #2399
  * #2410
* #2374
  * #2398
* #2404
  * #2405


### Testing instructions
* Sanity: Check that plugin updates work.
* Sanity: Check that license activation/deactivation works.
* Sanity: Check that notices appear correctly.
* More detailed testing steps in each pull request linked above.


### New/Updated Hooks

*


### Deprecated Code

*


### Screenshot / Video
